profile/README.md

```diff
@@ -6,8 +6,9 @@ Digital Asset empowers the world's most innovative organizations to work across
 
 ## Primary Digital Asset Open-Source Repositories
 
-[Daml](https://github.com/digital-asset/daml)
-[Canton](https://github.com/digital-asset/canton)
+[Daml](https://github.com/digital-asset/daml) is an open-source smart contract language for building future-proof distributed applications on a safe, privacy-aware runtime.
+
+[Canton](https://github.com/digital-asset/canton) is a Daml ledger interoperability protocol.
```

Comment:
This is not an accurate description of the repositories. I think I'd go more for something like:

```suggestion
Our primary open-source offering is Daml Open-Source, which is composed of the following components:

* [list of components]

These components are distributed across the following repositories:
* [digital-asset/daml](https://github.com/digital-asset/daml) contains the code for [compiler, IDE, HTTP, Triggers, Script, etc.]
* [digital-asset/canton](https://github.com/digital-asset/canton) contains the code for [participant, sequencer, ...]

For historical reasons, the fully assembled Daml Open-Source distribution is hosted as GitHub Releases on the [digital-asset/daml] repo. Note that this means releases on that repo include artifacts produced from other repositories.
```

Reply:
Thanks. I only copy/pasted the current description from the official [Daml](https://github.com/digital-asset/daml?tab=readme-ov-file#welcome-to-the-daml-repository) and [Canton](https://github.com/digital-asset/daml?tab=readme-ov-file#welcome-to-the-daml-repository) readme.
I'll work with marketing to cleanup these readme and then update the DA one accordingly.